mappings/net/minecraft/realms/SizeUnit.mapping

```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_4428 net/minecraft/realms/SizeUnit
+	FIELD field_20200 B Lnet/minecraft/class_4428;
+	FIELD field_20201 KB Lnet/minecraft/class_4428;
+	FIELD field_20202 MB Lnet/minecraft/class_4428;
+	FIELD field_20203 GB Lnet/minecraft/class_4428;
```

Comment:
Aren't these already auto-inferred by the enum automapper?

Reply:
Apparently not? I wouldn't have mapped them if they had names in Enigma.

Reply:
I looked now and they do indeed have implicit names. They didn't before. The code in Enigma looked like:

```
public static final class_4428 field_20200 = new class_4428("B", 0);
```